spec/v1/function-builder.spec.ts

```diff
@@ -472,11 +472,59 @@ describe('FunctionBuilder', () => {
     ).to.throw();
   });
 
-  it('', () => {
+  it('should throw an error if private identifier is in the invoker array', () => {
     expect(() =>
       functions.runWith({
         invoker: ['service-account1', 'private', 'service-account2'],
       })
     ).to.throw();
   });
+
+  it('should allow valid secret config expressed using short form', () => {
+    const secrets = ['API_KEY', 'API_KEY@3', 'API_KEY@latest'];
```

Comment:
Can we cut @version from scope? I'd like to not create more of a legacy burden than we need.

Reply:
Discussed offline. Not supporting versions in the first iteration.